### ISSUE TITLE
SESSION ERROR: SESSION: Your session has expired, please reauthenticate (9007)

### ISSUE THREAD

**Issue by Hamsterman** [linuxserver/docker-openvpn-as#116]
I have just created a fresh install of version 2.8.3.

I was able to login and change settings but after I restart the container I get this error:

SESSION ERROR: SESSION: Your session has expired, please reauthenticate (9007)

NET_ADMIN is enabled.

**Comment by j0nnymoe** [comment]
closing as dupe of https://github.com/linuxserver/docker-openvpn-as/issues/113

**Comment by xionous** [comment]
Found a possible solution to this issue for others who may still be having it.

In the database file located in the docker config folder openvpn-as/etc/db/config_local.db

row 6 "vpn.server.daemon.enable" is set to true, setting this to false gets rid of the session expired error and from what I can tell has no effect on the functionality of the VPN with this setting on false I can connect to my VPN externally with no issues. I am now running the latest version of the docker.

I found this by doing a comparison from an older version of this container 2.7.5-ls22 and in this version the DB variable is already set to false and functions fine with it set to false.
